templates/cloudflare-pages/package.json

```diff
@@ -20,7 +20,7 @@
     "react-dom": "^17.0.2"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^3.4.0",
+    "@cloudflare/workers-types": "^3.10.0",
```

Comment:
I think we should update `wrangler` dependency here as well?

Reply:
I will fix it.

Reply:
Fixed!